rotkehlchen/chain/zksync_lite/manager.py

```diff
@@ -860,4 +866,24 @@ def decode_undecoded_transactions(self, force_redecode: bool) -> int:
 
             self.decode_transaction(transaction, tracked_addresses)
 
+            if send_ws_notifications and tx_index % 10 == 0:
+                self.msg_aggregator.add_message(
+                    message_type=WSMessageType.EVMLIKE_UNDECODED_TRANSACTIONS,
+                    data={
+                        'evm_chain': self.ethereum_inquirer.chain_name,
+                        'total': len(transactions),
```

Comment:
define as a variable. I know that is O(1) but left prefers having the call outiside of the loop. Having them in python can have a big impact on performance